python/ray/state.py

```diff
@@ -385,6 +385,66 @@ def client_table(self):
 
         return _parse_client_table(self.redis_client)
 
+    def _job_table(self, job_id):
+        """Fetch and parse the job table information for a single job ID.
+
+         Args:
+            job_id: A job ID to get information about.
+         Returns:
+            A dictionary with information about the job ID in question.
+        """
+        # Allow the argument to be either a DriverID or a hex string.
```

Comment:
Did you mean `JobID`?

Reply:
fixed

Reply:
This isn't actually fixed in the comment above, right?

Reply:
yeah, the fix got rolled back when I force pushed